src/python/WMCore/ReqMgr/Web/tools.py

```diff
@@ -34,6 +31,31 @@
 except:
     pass
 
+
+def quote(data):
+    """
+    Sanitize the data using cgi.escape.
+    """
+    if  isinstance(data, (int, float)):
+        res = data
+    elif  isinstance(data, dict):
+        res = data
+    elif  isinstance(data, list):
+        res = data
+    elif  isinstance(data, (int, long, float)):
```

Comment:
Given all these if's have do the same thing, why don't we do a single `if ... else ...`? BTW, besides long, the last one is dup. 
Also, remove all these double space please. We follow the pep-8 convention and this will just create pylint complaints whenever someone change this file.